example/config.yml

```diff
@@ -1,5 +1,6 @@
 default:
   bucket: cumulus-test-sandbox-internal
+  apiUsername: mhuffnagle2
```

Comment:
I'll update it to "jasmine", and configure the list of allowed users in `example/cumulus-tf/main.tf` to include "jasmine" as well. Because of the way that our integration tests work, that does not need to be a real Earthdata Login user.